audioread/ffdec.py

```diff
@@ -73,10 +74,24 @@ def run(self):
 class FFmpegAudioFile(object):
     """An audio file decoded by the ffmpeg command-line utility."""
     def __init__(self, filename, block_size=4096):
+        # creation_flag default value
+        subprocess_flags = 0
+        # for windows, we set a custom creationflag to be passed to popen
+        # to avoid the Windows GPF dialog if the invoked program dies.
+        if sys.platform.startswith("win"):
+            # See comp.os.ms-windows.programmer.win32
+            # How to suppress crash notification dialog?, Jan 14,2004 -
+            # Raymond Chen's response [1]
```

Comment:
We don't need these comments from the Stack Overflow answer. Perhaps a link to the answer page would be helpful, though.